### ISSUE TITLE
Model inheritance with `primary_key` not working

### ISSUE THREAD

**Issue by derzinn** [collerek/ormar#253]
Playing around with Ormar, I found that model inheritance wasn't working as expected. 

```py
class BaseMeta(ormar.ModelMeta):
    database = database
    metadata = metadata


class BaseModel(ormar.Model):
    class Meta:
        abstract = True

    id: uuid.UUID = ormar.UUID(
        primary_key=True,
        default=uuid.uuid4,
        uuid_format="string"
    )

    created_at: datetime.datetime = ormar.DateTime(default=datetime.datetime.utcnow())
    updated_at: datetime.datetime = ormar.DateTime(default=datetime.datetime.utcnow())


class Member(BaseModel):
    class Meta(BaseMeta):
        tablename = "members"

    first_name: str = ormar.String(max_length=50)
    last_name: str = ormar.String(max_length=50)
```

If i run `await Member.objects.create(first_name="foo", last_name="bar")` i got the following error:

> asyncpg.exceptions.NotNullViolationError: null value in column "id" of relation "members" violates not-null constraint
> DETAIL:  Failing row contains (null, 2021-06-24 18:15:38.164825, 2021-06-24 18:15:38.164956, foo, bar).

How do I get this example working?

---

ormar              0.10.12
databases          0.4.3
psycopg2           2.9.1

**Comment by derzinn** [comment]
I don't know if that helps, but Alembic is properly generating the tables from the inherited model example.

```py
def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.create_table('members',
    sa.Column('id', sa.CHAR(36), nullable=False),
    sa.Column('created_at', sa.DateTime(), nullable=True),
    sa.Column('updated_at', sa.DateTime(), nullable=True),
    sa.Column('first_name', sa.String(length=50), nullable=False),
    sa.Column('last_name', sa.String(length=50), nullable=False),
    sa.PrimaryKeyConstraint('id')
    )
    # ### end Alembic commands ###
```

**Comment by collerek** [comment]
Fixed in 0.10.13 (via #254), please update and check again. 
Thanks for reporting!
